examples/libtorchaudio/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
If the goal is to show people how to make one themselves (educational), I'd recommend echoing the steps in the README as comments within the script to make it easier to follow along.

Reply:
thanks for the suggestion. let me follow on that one with another PR.